### PR TITLE
Output `cherry-pick` command in issue

### PR DESCRIPTION
**Pull request by dhaiducek**
I couldn't get the tests to run correctly locally, so here we go!

### CODE REVIEW

**Comment by dhaiducek** [comment]
Tests added--ready for review

File: src/github.ts [hunk 1]
```diff
@@ -57,7 +63,11 @@ export async function getRequiredChecks(
   repoName: string,
   branchName: string,
 ): Promise<Set<string>> {
-  const branch = await client.repos.getBranch({ owner: organization, repo: repoName, branch: branchName });
+  const branch = await client.repos.getBranch({
```
Comment (mprahl) [inline src/github.ts:66]: Is your auto-formatter set to a low line-length? I have mine set to 120 to match the linter. It'd be good if you could revert the formatting changes in this file.

File: src/github.ts [hunk 2]
```diff
@@ -39,6 +41,10 @@ export async function createFailureIssue(
     "issue is closed.\n\n" +
     "![sad Yoda](https://media.giphy.com/media/3o7qDK5J5Uerg3atJ6/giphy.gif)";
 
+  if (patchCmd) {
```
Comment (mprahl) [inline src/github.ts:44]: Could you put this comment before the gif?

**Comment by mprahl** [comment]
Thanks for the cool contribution @dhaiducek! Just a couple of minor comments.

**Review by mprahl — APPROVED** [review]

**Pull request merged**